src/benchmarks/micro/libraries/System.IO.Hashing/CrcAppendTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.IO.Hashing.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public abstract class CrcAppendTests<T>
```

Comment:
in the future we might add benchmarks for other methods, so we could make the type name more general
```suggestion
    public abstract class Crc_Perf<T>
```